### PR TITLE
Completed express-get-json exercise. -Lindsey Laveirge

### PR DESCRIPTION
**Pull request by llaveirge**
image

image

image

### CODE REVIEW

File: express-get-json/index.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    course: 'JavaScript',
+    score: 92
+  }
+};
+
+// Store grade objects in array:
+const gradesArr = [];
+for (var curId in grades) {
+  gradesArr.push(grades[curId]);
+}
```
Comment (scbowler) [inline express-get-json/index.js]: You want to do this inside the route handler, so if the grades object changes the client will always get the most up to date data.

**Review by scbowler — CHANGES_REQUESTED** [review]
On the right track just needs a little refactoring. Let me know if you have any questions.

**Comment by llaveirge** [comment]
New screenshots: 
image

image

image

**Review by scbowler — APPROVED** [review]
Great job implementing the changes, looks good! 👍

**Comment by llaveirge** [comment]
Changes completely made sense, just a brain compilation error.

**Pull request merged**